packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/CustomUserControlViewManagerCPP.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "winrt/Microsoft.ReactNative.Bridge.h"
+
+namespace winrt::SampleLibraryCPP::implementation {
+
+struct CustomUserControlViewManagerCPP
+    : winrt::implements<
+          CustomUserControlViewManagerCPP,
+          winrt::Microsoft::ReactNative::Bridge::IViewManager> {
+ public:
+  CustomUserControlViewManagerCPP() = default;
+
+  // IViewManager
+  winrt::hstring Name() noexcept;
+
```

Comment:
method is const - can it be marked as such?

Reply:
cppwinrt doesn't like it